aldryn_newsblog/tests/test_sitemaps.py

```diff
@@ -80,12 +80,6 @@ def test_languages_support(self):
         with override('de'):
             de_article = self.create_article()
 
-        multilingual_sitemap = NewsBlogSitemap()
-        self.assertArticlesIn(
-            [multilanguage_article, en_article, de_article],
-            multilingual_sitemap
-        )
-
```

Comment:
I think I was just thinking that this I18NSitemap is now in translation-tools and is tested there. We can keep it here too though.